mobile/src/helpers/health.ts

```diff
@@ -0,0 +1,86 @@
+import GoogleFit, { Scopes } from 'react-native-google-fit';
+import AppleHealthKit from 'rn-apple-healthkit';
+import logger from 'common/logger';
+
+const runOptions = {
+  scopes: [
+    Scopes.FITNESS_ACTIVITY_READ,
+    Scopes.FITNESS_ACTIVITY_WRITE,
+    Scopes.FITNESS_BODY_READ,
+  ]
+};
+
+const options = {
+  permissions: {
+    read: ["StepCount", "BiologicalSex", "DateOfBirth"],
+    write: [],
+  },
+};
+
+export const auth = async() => {
+  var isAuth;
+
+  return await GoogleFit.checkIsAuthorized().then (() => {
```

Comment:
are the comments in this method needed? They look like old logic that is no longer being used